lib/avo/fields/select_field.rb

```diff
@@ -55,6 +56,15 @@ def label
         display_value ? value : options.invert[value]
       end
 
+      def to_permitted_param
+        case id
+          when :sizes
+            {"#{id}": []}
+          else
+            id
+          end
```

Comment:
Let's perform the conditional check on `@multiple`. Right now it will only apply to fields named `sizes`.

```suggestion
        return id unless @multiple

        {"#{id}": []}
```